specs/address_book_list/01_main.rst

```diff
@@ -0,0 +1,79 @@
++--------+------------------+
+| issue  | title            |
++--------+------------------+
+| 81_    | Address Book List|
++--------+------------------+
+
+.. sectnum::
+
+.. _81: https://github.com/gnosis/safe/issues/81
+
+
+Address Book List
+=================
+
+Problem Definition
+-------------------------------
+
+- User can recognize names better than full addresses
+- User wants to associate addresses to names
+- User can see the address book entries and their details
+- The address book is not synced (ie.: it is part of the local storage only)
+
+Inputs
+-----------
+
+- Address book entries stored locally (local database) 
+
+Outputs
+------------
+
+- Formatted list to be displayed to the user. Each entry has:
+    - Identicon
+    - Name
+    - Ethereum address
+
+Use Case Scenarios
+-----------------------
+
+Happy Case
+~~~~~~~~~~~~~~~~~
+
+.. _happy_case_preconditions:
+
+Preconditions
```

Comment:
There is nothing else you can do in this list?
- Select entry to edit/view/remove (this could link to a different spec, but the interaction should be mentioned here)
- How are the addresses sorted? Can the sorting be changed? 
- Do we have a search here (this could again link to a different spec)?

Reply:
Will add the view/add/edit options here and then link once the other specs are ready.

Reply:
Updated the use cases. Will add the link once the other specs are finished.